persistence/mongo/src/main/scala/za/co/absa/spline/persistence/mongo/dao/LineageDAOv4.scala

```diff
@@ -314,12 +329,6 @@ object MutableLineageUpgraderV4 {
       upgradeDataTypeOf(expr)
       upgradeTypeHintOf(expr)
     }
-
-    def renameExpressionType(expr: DBObject, typeName: String): Unit =
```

Comment:
Was it necessary to move the methods up? It might just complicate further merge to 0.4 with no gain.

Reply:
It was necessary - When i added new code it caused some "forward reference" compiler error. I will try it move it back since we have changed implementation during code review.